### PR TITLE
Redis: add generated config docs and clean up `Getting Started`

### PR DESCRIPTION
**Pull request by laysauchoa**
# What changed, and why it matters

Clean up getting started page
Generated config docs for Redis

### CODE REVIEW

File: docs/products/redis/get-started.rst [hunk 1]
```diff
@@ -1,59 +1,38 @@
-Get Started with Aiven for Redis
+Get started with Aiven for Redis
 ================================
 
-Get up and running with Aiven for Redis. This article shows you how to set up a Redis service in Aiven and learn more about the platform.
+Aiven for Redis services are managed from the `Aiven
+Console <https://console.aiven.io/>`__ .
```
Comment (anelook) [inline docs/products/redis/get-started.rst:5]:
```suggestion
Console <https://console.aiven.io/>`__.
```

File: docs/products/redis/get-started.rst [hunk 2]
```diff
@@ -1,59 +1,38 @@
...
 
-Create a new Aiven service
---------------------------
+You can start an Aiven for Redis service by following the steps laid out
```
Comment (anelook) [inline docs/products/redis/get-started.rst:7]: This and previous paragraph seem to be small, let's combine them?

File: docs/products/redis/get-started.rst [hunk 3]
```diff
@@ -1,59 +1,38 @@
...
+in :doc:`this article </docs/platform/howto/create_new_service>`.
 
-To create a new Aiven for Redis service:
+The view returns to the service list, where the new service is shown
```
Comment (anelook) [inline docs/products/redis/get-started.rst:10]: I feel it is a bit abrupt. You assume that the user followed the steps, but the instruction to follow the steps was a suggestion. I'd rephrase it. Something like this - 

Once the service creation is initiated, you can go to its dedicated service page .

File: docs/products/redis/get-started.rst [hunk 4]
```diff
@@ -1,59 +1,38 @@
...
 
 Next steps
 ----------
 
-Code examples for connecting to Redis from your application:
-   * :doc:`Go <howto/connect-go>`
-   * :doc:`Node <howto/connect-node>`
-   * :doc:`PHP <howto/connect-php>`
-   * :doc:`Python <howto/connect-python>`
+* Learn how to connect to Aiven for Redis:
```
Comment (anelook) [inline docs/products/redis/get-started.rst:29]:
```suggestion
* Learn how to connect to Aiven for Redis by using different programming languages:
```

File: docs/products/redis/get-started.rst (hunk 2 above)
Comment (anelook) [inline docs/products/redis/get-started.rst:7]: You're relying later that users will follow the instructions, I'd omit "You can " and use "Create an Aiven for Redis service by following the steps laid out"

**Review by anelook — COMMENTED** [review]
Cool content! Thx for cleaning up and adding references to properties!

I added some of the comments/suggestions.

**Comment by laysauchoa** [comment]
thank you for the suggestions! they have been addressed here: https://github.com/aiven/devportal/pull/436/commits/64ff7cdd047ed1048871cc5177f86f73263a3996

File: docs/products/redis/get-started.rst [hunk 5]
```diff
@@ -1,59 +1,35 @@
-Get Started with Aiven for Redis
+Get started with Aiven for Redis
 ================================
 
-Get up and running with Aiven for Redis. This article shows you how to set up a Redis service in Aiven and learn more about the platform.
+Aiven for Redis services are managed from the `Aiven
+Console <https://console.aiven.io/>`__. Create an Aiven for Redis service by following the steps laid out in :doc:`this article </docs/platform/howto/create_new_service>`.
```
Comment (anelook) [inline docs/products/redis/get-started.rst:5]:
```suggestion
Console <https://console.aiven.io/>`__. Create a new Aiven for Redis service by following the steps laid out in :doc:`this article </docs/platform/howto/create_new_service>`.
```

File: docs/products/redis/get-started.rst [hunk 6]
```diff
@@ -1,59 +1,35 @@
...
 
-Create a new Aiven service
---------------------------
+Once the service creation is initiated, you can go to its dedicated service page where the new service is shown
+with an indicator that it is being created.
 
-To create a new Aiven for Redis service:
+Click the service name in the list to go to the "**Overview**" page.
```
Comment (anelook) [inline docs/products/redis/get-started.rst:10]: Because of the previous sentence we're already in the service page, so we don't have a list, right?
You can replace it with something like "The **Overview** tab shows connection...." and continue with your paragraph :)

**Review by anelook — CHANGES_REQUESTED** [review]
Thanks for making the changes, I've added a couple of small things to make text a biyt more consistent.

**Review by anelook — CHANGES_REQUESTED** [review]
Looks great! One last small thing - we usually don't use quotes around elements (for example **Overview**, *Advanced configuration*, etc.). Let's remove them. You can look for similar entries in existing articles for a reference :)

**Comment by laysauchoa** [comment]
> Looks great! One last small thing - we usually don't use quotes around elements (for example **Overview**, _Advanced configuration_, etc.). Let's remove them. You can look for similar entries in existing articles for a reference :)

I was checking this: https://developer.aiven.io/docs/products/mysql/get-started.html should we remove there as well?

**Comment by anelook** [comment]
> > Looks great! One last small thing - we usually don't use quotes around elements (for example **Overview**, _Advanced configuration_, etc.). Let's remove them. You can look for similar entries in existing articles for a reference :)
> 
> I was checking this: https://developer.aiven.io/docs/products/mysql/get-started.html should we remove there as well?

I believe so - in majority of cases we don't use quotes, because formatting already gives a separation.

**Review by anelook — APPROVED** [review]
